### PR TITLE
Added WebPortal children to each client type

### PR DESCRIPTION
**Pull request by danigargar**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Review by ironArt3mis — CHANGES_REQUESTED** [review]
Functional review required

File: web/portal/brand/cypress/e2e/Clients/Residential/Residentials.cy.js [hunk 1]
```diff
@@ -32,4 +32,9 @@ describe('in Residential Companies', () => {
   // DELETE
   ///////////////////////
   it('delete Residential Companies', deleteCompany);
+
+  ///////////////////////
+  // WEB PORTALS
+  ///////////////////////
```
Comment (rbatistadev) [inline web/portal/brand/cypress/e2e/Clients/Residential/Residentials.cy.js:38]:
```suggestion
  ///////////////////////
  // POST
  ///////////////////////
```

File: web/portal/brand/cypress/e2e/Clients/Retails/Retails.cy.js [hunk 2]
```diff
@@ -31,4 +32,9 @@ describe('in Residential Companies', () => {
   // DELETE
   ///////////////////////
   it('delete Retail Companies', deleteCompany);
+
+  ///////////////////////
+  // WEB PORTALS
+  ///////////////////////
```
Comment (rbatistadev) [inline web/portal/brand/cypress/e2e/Clients/Retails/Retails.cy.js:38]:
```suggestion
  ///////////////////////
  // POST
  ///////////////////////
```

File: web/portal/brand/cypress/e2e/Clients/VirtualPbxs/VirtualPbxs.cy.js [hunk 3]
```diff
@@ -28,4 +28,9 @@ describe('in Companies', () => {
   // DELETE
   ///////////////////////
   it('delete Virtual Pbx Companies', deleteCompany);
+
+  ///////////////////////
+  // WEB PORTALS
+  ///////////////////////
```
Comment (rbatistadev) [inline web/portal/brand/cypress/e2e/Clients/VirtualPbxs/VirtualPbxs.cy.js:34]:
```suggestion
  ///////////////////////
  // POST
  ///////////////////////
```

**Review by ironArt3mis — APPROVED** [review]

**Pull request merged**